tests/queries/0_stateless/02346_text_index_function_hasAnyAllTokens.sql

```diff
@@ -92,11 +92,11 @@ SELECT id FROM tab WHERE hasAllTokens('a b', 'a c');
 SELECT id FROM tab WHERE hasAllTokens(col_str, 'a a');
 SELECT id FROM tab WHERE hasAllTokens(col_str, 'b c');
 
--- Test search without needle on non-empty columns (all are expected to match)
+-- Test search without needle on non-empty columns (all are expected to match nothing)
 SELECT count() FROM tab WHERE hasAnyTokens(col_str, []);
 SELECT count() FROM tab WHERE hasAllTokens(col_str, []);
-SELECT count() FROM tab WHERE hasAnyTokens(col_str, ['']); -- matches nothing
-SELECT count() FROM tab WHERE hasAnyTokens(col_str, ''); -- TODO currently this goes through the default tokenizer and matches everything
+SELECT count() FROM tab WHERE hasAnyTokens(col_str, ['']);
```

Comment:
Maybe you could add:

`SELECT count() FROM tab WHERE hasAnyTokens(col_str, ['','']);`

I recently found a corner case for this case